src/modules/flight_mode_manager/tasks/Utility/StickAccelerationXY.cpp

```diff
@@ -157,7 +159,13 @@ Vector2f StickAccelerationXY::calculateDrag(Vector2f drag_coefficient, const flo
 
 	// increase drag with squareroot function when velocity is lower than 1m/s
 	const Vector2f velocity_with_sqrt_boost = vel_sp.unit_or_zero() * math::sqrt_linear(vel_sp.norm());
-	return drag_coefficient.emult(velocity_with_sqrt_boost);
+
+	// only apply the drag increase below 1m/s when actually brakeing such that speeds below 1m/s
```

Comment:
```suggestion
	// only apply the drag increase below 1m/s when actually braking such that speeds below 1m/s
```

Reply:
Did you get complaints that the braking was too hard? This will change the braking behavior for everyone, right?

Reply:
Thanks for checking 🙏  I'm fixing the typo.
No the goal of this is to not change the braking behavior but to not map to a different speed if you want to fly below 1m/s in the first place (without braking).